### ISSUE TITLE
Freeze in core / loss of focus opening Word (365 at least) in NVDA 2020.4

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#12124]
### Steps to reproduce:

I've had several reports of this.  One gave the following steps, which seems to work to reproduce the issue for me:

1.       Open File explorer
2.       Navigate to the desired document
3.       Document is selected
4.       NVDA starts reading information
5.       While it’s reading, ENTER is pressed to open the file
6.       Speech stops
7.       File Loads
8.       NVDA is quiet until ALT+TAB is pressed to switch focus back and forth.

### Actual behavior:

As noted, the file loads in Word, but NVDA stops responding until focus is moved (eg alt+tab away and back, or simply alt to the ribbon and alt to close back to the document).

Log attached.  See from line 169, up arrow to document, then enter before it is read, followed by core frozen in stack.  In this log I pressed NVDA+F1 to get the object when focus was lost, but pressing alt twice or otherwise moving focus away from the document window and back does restore NVDA.

[wordfreezelog.txt](https://github.com/nvaccess/nvda/files/6089372/wordfreezelog.txt)

### Expected behavior:

NVDA should be responsive once the document loads without needing to move focus.

#### NVDA version:

NVDA 2020.4 installed.

#### Windows version:

Windows 10 (64-bit) Version: 2009, Build: 19042 
 

#### Name and version of other software in use when reproducing the issue:

Office 365 (64-bit) Version: 16.0.13801.20214 

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

This has only been reported in the last couple of weeks, but in testing the above scenario just now, I can reproduce back to 2016.1 (I didn't test earlier versions).  So likely a recent change in Office.

#### If add-ons are disabled, is your problem still occurring?

No add-ons installed.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes, no change.

**Comment by Adriani90** [comment]
This is a duplicate of 8389. Closing as duplicate.

**Comment by Adriani90** [comment]
#8389
